README.md

```diff
@@ -411,6 +411,7 @@ Table of Contents
   * [atomist.com](https://atomist.com/) — A quicker and more convenient way to automate a variety of development tasks. Now in beta.
   * [auth0.com](https://auth0.com/) — Hosted free for development SSO. Up to 2 social identity providers for closed-source projects.
   * [Authress](https://authress.io/) — Authentication login and access control, unlimited identity providers for any project. Facebook, Google, Twitter and more. First 1000 API calls are free.
+  * [Authy](https://authy.com) - Two-factor authentication (2FA) on multiple devices, with backups. Drop-in replacement for Google Authenticator.
```

Comment:
Please add something like "free for less than 100 authentications a month"

Reply:
@ripienaar I have added that information in 0742940a374766ea3bc1a87b2ea9226be816ae21